<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>8</y>
      <width>305</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="InputField">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="InputLabel">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Input:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="InputEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="InputButton">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>48</y>
      <width>145</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="ConvertField">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ConvertEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ConvertLabel">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Convert to:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ConvertButton">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>80</y>
      <width>128</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
